keyboards/keebio/iris/keymaps/osiris/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include "iris.h"
```

Comment:
Lines 1-3 should be replaced with:

```c
#include QMK_KEYBOARD_H
```

which will include all of these files.

Reply:
updated code